docs/infrastructure/README.md

```diff
@@ -0,0 +1,62 @@
+# Overview
+
+Student Robotics has a variety of infrastrcture in order to support its mission.
```

Comment:
```suggestion
Student Robotics has a variety of infrastructure in order to support its goals.
```

Reply:
Typo fixed in 24fe613712cbad13a58da8fedcec70ab48797726. Is there a reason to change mission to goals?